Resources/public/js/views/ez-locationviewview.js

```diff
@@ -31,6 +31,7 @@ YUI.add('ez-locationviewview', function (Y) {
                 Y.Array.each(this.get('tabs'), function (t) {
                     t.set('active', this.get('active'));
                 }, this);
+                window.scrollTo(0,0);
```

Comment:
as far as I get it, it's not the right way (tm) to fix the scroll issue. First, this should be made once and for all views not only the LocationView. Second, you should probably take example on the YUI code doing that (see https://yuilibrary.com/yui/docs/api/files/pjax_js_pjax-base.js.html#l305)